src/shogun/kernel/GaussianKernel.h

```diff
@@ -162,15 +129,10 @@ class GaussianKernel: public ShiftInvariantKernel
 	 */
 	virtual void load_serializable_post() noexcept(false);
 
+	virtual SGMatrix<float64_t>
+	sample_spectral_density(int32_t dim_output, int32_t dim_input_space) const;
```

Comment:
so would this become purely virtual at some point?

Reply:
this method should override a virtual method in the kernel's base class

Reply:
But we dont want to force devvs to implement the spectral density for all shift invariant kernels